src/lib/logger.ts

```diff
@@ -0,0 +1,19 @@
+import winston, { format } from 'winston'
+
+const { combine, timestamp, prettyPrint } = format
+
+export const createBaseLogger = (path: string) => {
+  const formats = [
+    winston.format.json(),
+    timestamp(),
+    process.env.NODE_ENV === 'development' ? prettyPrint() : undefined,
+  ].filter((v) => !!v) as winston.Logform.Format[]
+
+  return winston.createLogger({
+    format: combine(...formats),
+    transports: [new winston.transports.Console()],
+    defaultMeta: {
```

Comment:
`winston` also has a bunch of extra functions such as `rejectionHandlers` and `exceptionHandlers`, would they be useful in our context as a catch-all?

Reply:
All exceptions should be caught on the onError handler, im not sure what value these add so will leave it for future prs if u want to add it!